cli/src/complete.rs

```diff
@@ -28,19 +29,42 @@ use crate::config::default_config;
 use crate::config::LayeredConfigs;
 use crate::ui::Ui;
 
+static BOOKMARK_HELP_TMPL: &str = r#"
```

Comment:
nit: `const`? (it wouldn't matter, though)

If you like, this can be a TOML document to be installed by `--config-toml`, and can be referred to as a template alias function.